Library/Formula/sip.rb

```diff
@@ -13,17 +14,15 @@ class Sip < Formula
     sha256 "8832546d36baa62fdecd0df427ba4f3b02ab2f39fc5fcb47f114ae5020f11342" => :mountain_lion
   end
 
-  head "https://www.riverbankcomputing.com/hg/sip", :using => :hg
-
   option "without-python", "Build without python2 support"
   depends_on :python => :recommended if MacOS.version <= :snow_leopard
   depends_on :python3 => :optional
 
-  if build.without?("python3") && build.without?("python")
-    odie "sip: --with-python3 must be specified when using --without-python"
-  end
-
   def install
+    if build.without?("python3") && build.without?("python")
+      raise "sip: --with-python3 must be specified when using --without-python"
```

Comment:
I know, see https://github.com/Homebrew/homebrew/pull/44954#discussion_r42487584 and the ensuing discussion.